docs/concepts/package-installation-process.md

```diff
@@ -0,0 +1,49 @@
+---
+title: What happens when a package is installed?
+description: Detailed information about the package installation process
+author: karann-msft
+ms.author: karann
+ms.date: 0/20/2019
+ms.topic: conceptual
+---
+
+# What happens when a NuGet package is installed?
+
+Simply said, the different NuGet tools typically create a reference to a package in the project file or `packages.config`, then perform a package restore, which effectively installs the package. The exception is `nuget install`, which only expands the package into a `packages` folder and does not modify any other files, unless you specify options such as `-OutputDirectory` or `packages.config` as value for the `-ConfigFile` option.
```

Comment:
Yes, when I reread the description yesterday, that's what it sounded like to me, too. My plan right now is to try to test Nikolche's comments. I just don't see any example at the moment that shows an install command that would also modify csproj. I see the example using packages.config, but I'm not sure whether that example works as intended.

Reply:
I reverted these changes to match content that is currently live. I wasn't able to make any changes to the csproj using install. I did include an example instruction for packages.config, which installs the package locally based on packages.config. But this still does not seem to modify the csproj. @nkolev92 in case I'm missing something.

Reply:
Also, @karann-msft FYI

Reply:
I tested it and the latest doesn't. 
Seems to have been removed.